### PR TITLE
#165413111 user is able to like and dislike an article

### PR DESCRIPTION
**Pull request by Kabalisa**
#### Description
In our authors' haven project we value would like to appreciate authors who write good articles wit 
h a like and encourage those who have to put in the effort with a dislike. this feature will add that functionality to our project.
#### Type of change
N/A
#### How Has This Been Tested?
this has been tested by sending request specified in the readme to these endpoints
*  create a like on an article ```POST /api/articles/:slug/likes/like```
*  create a dislike on an article ```POST /api/articles/:slug/dislikes/dislike```
* fetch all likes on an article ```GET /api/articles/: slug/likes```
* fetch all dislikes on an article ```GET /api/articles/:slug/dislikes```
#### Checklist:
N/A
#### Pivotal Tracker ID
[#165413112](https://www.pivotaltracker.com/story/show/165413112)

### CODE REVIEW

**Comment by Niyitangasam** [comment]
Update this branch and resolve conflicts

**Review by ericnyirimana — APPROVED** [review]

File: helpers/article.js [hunk 1]
```diff
@@ -209,5 +209,137 @@ class ArticleHelper {
...
+     * @param {object} res - an object
+     * @param {object} next - an object
+     * @return {object} Returns response
+     * @static
+     */
+  static async isExisting(req, res, next) {
+    const { slug } = req.params;
+    const { id } = req.user;
+    const { option } = req.params;
+    // const DISLIKE = req.params.dislike;
```
Comment (AJAkimana) [inline helpers/article.js]: Delete this comment

**Review by jabichris — APPROVED** [review]

**Review by jabichris — APPROVED** [review]
I approve but you need to update your branch

**Review by AJAkimana — COMMENTED** [review]
Great work. Check the comments I left

**Pull request merged**